doc/api/axes_api.rst

```diff
@@ -1,12 +1,668 @@
+
+
+   Axes.can_pan
+   Axes.can_zoom
+
+   Axes.get_navigate
+   Axes.set_navigate
+   Axes.get_navigate_mode
+   Axes.set_navigate_mode
+
+   Axes.start_pan
+   Axes.drag_pan
+   Axes.end_pan
+
+   Axes.format_coord
+   Axes.format_cursor_data
+   Axes.format_xdata
+   Axes.format_ydata
+
+   Axes.hitlist
+   Axes.mouseover
+   Axes.in_axes
+
+   Axes.pick
+   Axes.pickable
+   Axes.get_picker
+   Axes.set_picker
+
+   Axes.set_contains
+   Axes.get_contains
+
+   Axes.contains
+   Axes.contains_point
+
+   Axes.get_cursor_data
+   Axes.get_cursor_props
+   Axes.set_cursor_props
+
+Children
+========
+
+.. autosummary::
+   :toctree: _as_gen
+   :nosignatures:
+
+   Axes.get_children
+   Axes.get_images
+   Axes.get_lines
+   Axes.findobj
+
+
+Drawing
+=======
+
+.. autosummary::
+   :toctree: _as_gen
+   :nosignatures:
+
+   Axes.draw
+   Axes.draw_artist
+   Axes.redraw_in_frame
+   Axes.get_renderer_cache
+
+   Axes.get_rasterization_zorder
+   Axes.set_rasterization_zorder
+
+   Axes.get_window_extent
+   Axes.get_tightbbox
+
+
+Bulk property manipulation
+==========================
+
+.. autosummary::
+   :toctree: _as_gen
+   :nosignatures:
+
+   Axes.set
+   Axes.update
+   Axes.properties
+   Axes.update_from
+
+
+General Artist Properties
+=========================
+
+.. autosummary::
+   :toctree: _as_gen
+   :nosignatures:
+
+   Axes.set_alpha
+   Axes.set_animated
+   Axes.set_clip_box
+   Axes.set_clip_on
+   Axes.set_clip_path
+   Axes.set_gid
+   Axes.set_label
+   Axes.set_url
+   Axes.set_visible
+   Axes.set_zorder
+   Axes.set_rasterized
+   Axes.set_sketch_params
+   Axes.set_agg_filter
+   Axes.set_snap
+   Axes.set_transform
+   Axes.set_path_effects
+
+   Axes.get_agg_filter
+   Axes.get_sketch_params
+   Axes.get_alpha
+   Axes.get_animated
+   Axes.get_clip_box
+   Axes.get_clip_on
+   Axes.get_clip_path
+   Axes.get_gid
+   Axes.get_label
+   Axes.get_url
+   Axes.get_visible
+   Axes.get_zorder
+   Axes.get_rasterized
+   Axes.get_transform
+   Axes.get_snap
+   Axes.get_path_effects
+
+
+   Axes.axes
+   Axes.get_axes
+   Axes.set_axes
+   Axes.set_figure
+   Axes.get_figure
+
+Artist Methods
+==============
+
+.. autosummary::
+   :toctree: _as_gen
+   :nosignatures:
+
+   Axes.is_figure_set
+   Axes.remove
+   Axes.is_transform_set
+
+
+Projection
+==========
+
+Methods used by `~matplotlib.axis.Axis` that must be overridden for
+non-rectilinear Axes.
+
+.. autosummary::
+   :toctree: _as_gen
+   :nosignatures:
+
+   Axes.name
+   Axes.get_xaxis_transform
+   Axes.get_yaxis_transform
+   Axes.get_data_ratio
+   Axes.get_data_ratio_log
+
+   Axes.get_xaxis_text1_transform
+   Axes.get_xaxis_text2_transform
+   Axes.get_yaxis_text1_transform
+   Axes.get_yaxis_text2_transform
+
+
+Other
+=====
+
+.. autosummary::
+   :toctree: _as_gen
+   :nosignatures:
+
+   Axes.zorder
+   Axes.aname
+   Axes.get_default_bbox_extra_artists
+   Axes.get_transformed_clip_path_and_affine
+   Axes.has_data
+   Axes.hold
```

Comment:
Maybe this is the time to add a "Future Deprecation Warning" to the docstring in view of #3070